app/(gcforms)/[locale]/(support)/unlock-publishing/components/client/Success.tsx

```diff
@@ -1,9 +1,10 @@
-import { serverTranslation } from "@i18n";
+"use client";
```

Comment:
The "use client" is not required because `UnlockPublishingForm` is already a Client Component.